.github/workflows/docs.yml

```diff
@@ -32,3 +32,9 @@ jobs:
 
       - name: Lint (docs)
         run: pnpm lint:docs
+
+      - name: Build Nuxt
+        run: pnpm build
+
+      - name: Typecheck (docs)
+        run: pnpm typecheck:docs
```

Comment:
Maybe there is an opportunity here to move docs workflow back into main ci.yml but use `tj-actions/changed-files` instead to granularly enable workflows.